.github/workflows/kubernetes.yml

```diff
@@ -74,7 +74,7 @@ jobs:
       - name: Clean cluster
         if: always()
         run: |
-          kubectl delete -f docker/zefchain-k8s.yml
+          kubectl delete -f docker/local-tests/zefchain-k8s.yml
```

Comment:
Maybe a good time to change the `zefchain` occurence to `linera`?

Reply:
Good point lol